docassemble/EFSPIntegration/data/questions/admin_interview.yml

```diff
@@ -569,13 +680,14 @@ code: |
   get_global_payment_account = True
 ---
 code: |
+  # TODO(qs): this interacts with TOGA--not implemented
```

Comment:
This is actually implemented, on the Java side at least.  It just expects a JSON object with `"active": bool, "account_name": "myName"}`